content/funding-your-training.md

```diff
@@ -74,20 +74,31 @@
     - Isle of Man
 ---
 
+
+## Tuition fee and maintenance loans
+
+You can apply for a tuition fee loan of up to £9,250 to cover your teacher training, so you don't need to pay course fees upfront. You can also apply for a maintenance loan of up to £12,382 to help with living costs.
+
+You can still apply for a tuition fee and/or a maintenance loan if you already have a student loan, and regardless of whether you get a bursary or scholarship.
```

Comment:
Would 'or' make sense here instead of 'and/or'?

Reply:
I actually think 'and' is more appropriate. I believe most people will get both of these as opposed to just one.